docs/end-user-topics/dois.rst

```diff
@@ -0,0 +1,89 @@
+
+In Dockstore, the concept DOI is associated with the entry as a whole, and version DOIs are associated with entry versions.
+
+Dockstore Support for DOIs
+--------------------------
+
+Dockstore can generate DOIs for entries, as well as recognize some DOIs generated outside of Dockstore. Dockstore displays DOIs associated with entries. If you have multiple concept DOIs associated
+with a single entry, you can specify which DOI to display.
+
+Dockstore generates DOIs for entries 2 ways:
+
+* Explicitly initiated by a Dockstore user. This approach requires you linking your Zenodo account. You select a version via the Dockstore UI and click to create a DOI. See :doc:`Creating Snapshots & Requesting DOIs </advanced-topics/snapshot-and-doi>` for more details
+* Dockstore can automatically generate DOIs for your entries without you needing to link your Zenodo account. These DOIs are created in the Dockstore Zenodo Community
+
+In addition, Dockstore can discover DOIs created by a Zenodo-GitHub integration.
+
+Choosing which DOI to display
+-----------------------------
+
+When viewing the public page of an entry, the entry's concept DOI and the selected version's DOI are displayed.
+
+.. figure:: /assets/images/docs/show-doi.png
+   :alt: Show DOIs
+
+Dockstore only displays one DOI associated with a entry, even if there is more than one DOI, e.g., one generated by GitHub-Zenodo integration, and another generated by Dockstore.
+If this situation arises, you can control which DOI is displayed via your dashboard. Select the entry, then click "Manage DOIs", and you'll get a dialog like this. Select your choice
+and click "Save".
+
+   
+.. figure:: /assets/images/docs/manage-dois.png
+   :alt: Manage DOIs
+
+
+Features of the different ways of issuing DOIs
+----------------------------------------------
+
+With the different ways Dockstore supports DOIs, it may be confusing as to how to generate DOIs for your Dockstore entries.  Here's a high level overview of some of the features of each.
+
+
+Creating Snapshots & Requesting DOIs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+See :doc:`Creating Snapshots & Requesting DOIs </advanced-topics/snapshot-and-doi>` for more details
+
+* You must initiate the DOI creation for each version
+    * More control over when DOIs are created
+    * You have to individually do it for every tag
+* Version on Dockstore is snapshotted (frozen)
+* Requires a Zenodo account that you link to your Dockstore account
+
+Automatic Dockstore DOI Generation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+    This feature is currently only available on request. Please contact the Dockstore team at `discuss.dockstore.org <https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506>`__ if you want to participate.
+
+* Automatically mints DOIs for all Git tags pushed to GitHub repositories that have a corresponding entry in Dockstore
+* The DOIs are minted in the Zenodo Dockstore community
```

Comment:
The DOIs when users manually create it are also added to the Zenodo Dockstore community

Reply:
I did not know that.